From: Enrico Zini <enrico@debian.org>
Subject: remove "quiet" from the args passed to the server adapter
Forwarded: no

---
 bottle.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- python-bottle.orig/bottle.py
+++ python-bottle/bottle.py
@@ -1225,7 +1225,7 @@ def run(app=None, server=AutoServer, hos
         interval=1, reloader=False, **kargs):
     """ Runs bottle as a web server. """
     app = app if app else default_app()
-    quiet = bool(kargs.get('quiet', False))
+    quiet = bool(kargs.pop('quiet', False))
     # Instantiate server, if it is a class instead of an instance
     if isinstance(server, type):
         server = server(host=host, port=port, **kargs)
